lib/cdo/scripts/script_info.rb

```diff
@@ -0,0 +1,54 @@
+# Info on existing tutorial scripts on Code Studio,
+# sometimes referenced in Pegasus.
+# Used for conditional behaviors.
+
+class ScriptInfo
```

Comment:
maybe name this ScriptConstants and move to top level /lib/cdo to match ActivityConstants and WorkshopConstants